cclib/method/volume.py

```diff
@@ -221,9 +221,9 @@ def wavefunction(ccdata, volume, mocoeffs):
     wavefn.data = numpy.zeros(wavefn.data.shape, "d")
 
     conversion = convertor(1, "bohr", "Angstrom")
-    x = numpy.arange(wavefn.origin[0], wavefn.topcorner[0] + wavefn.spacing[0], wavefn.spacing[0]) / conversion
-    y = numpy.arange(wavefn.origin[1], wavefn.topcorner[1] + wavefn.spacing[1], wavefn.spacing[1]) / conversion
-    z = numpy.arange(wavefn.origin[2], wavefn.topcorner[2] + wavefn.spacing[2], wavefn.spacing[2]) / conversion
+    x = numpy.arange(wavefn.origin[0], wavefn.topcorner[0] + 0.5*wavefn.spacing[0], wavefn.spacing[0]) / conversion

```

Comment:
Can you factor this back out into a constant?

Reply:
I've just factored this out as a constant. Thanks for the comment!